ndcube/mixins/plotting.py

```diff
@@ -157,7 +185,23 @@ def _animate_cube(self, wcs, plot_axes=None, axes_coordinates=None,
             for axis_unit, coord_name in zip(axes_units, wcs.world_axis_physical_types):
```

Comment:
Isn't `axes_units` entered in array order?  If so, are they reversed relative to `wcs.world_axis_physical_types`?

Reply:
yes, but the utils function flips them into world order so all the plot methods here have them in WCSAxes order.